bin/realpath.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
I think we can drop this file since it isn't used

Reply:
Oops should have been removed.